index.html

```diff
@@ -117,7 +117,19 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
-(tryout.oninput = e => {
+let url = new URL(location);
+const selector = url.searchParams.get("selector");
+
+if (selector) {
+	selectorText.value = selector;
+	url.searchParams.set("selector", selectorText.value);
```

Comment:
I just noticed this, sorry if it was there before.
You are getting the selector URL param, setting the input to that if it exists, and then setting the selector URL param to the value of the input... which was the URL param you just got! So you're setting it to itself. Unless I'm missing something, this line can just be deleted.